### PR TITLE
[hi] Localize content/hi/bare_metal_machine.md

### PR DESCRIPTION
**Pull request by anubhav06**
This PR localizes the file `content/hi/bare_metal_machine.md` in Hindi

Ref: #270 

@anubha-v-ardhan @ @Garima-Negi Pls review

### CODE REVIEW

File: content/hi/bare_metal_machine.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: बेयर धातु मशीन
```
Comment (jayesh-srivastava) [inline content/hi/bare_metal_machine.md:2]: मेटल should be here as well.

Comment (jayesh-srivastava) [inline content/hi/bare_metal_machine.md:2]: मेटल should be here as well.

File: content/hi/bare_metal_machine.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+बेयर मेटल एक भौतिक कंप्यूटर को संदर्भित करता है, विशेष रूप से एक सर्वर, जिसमें एक, और केवल एक, ऑपरेटिंग सिस्टम होता है। आधुनिक कंप्यूटिंग में अंतर महत्वपूर्ण है क्योंकि कई सर्वर [वर्चुअल मशीन](/virtual_machine/) हैं। एक भौतिक सर्वर आमतौर पर एक काफी बड़ा कंप्यूटर होता है जिसमें शक्तिशाली हार्डवेयर अंतर्निहित होता है। [वर्चुअलाइज़ेशन](/virtualization/) के बिना किसी ऑपरेटिंग सिस्टम को स्थापित करना और सीधे उस भौतिक हार्डवेयर पर एप्लिकेशन चलाना, "बेयर मेटल" पर चलाने के रूप में जाना जाता है।
```
Comment (jayesh-srivastava) [inline content/hi/bare_metal_machine.md:9]: - क्योंकि कई सर्वर, [वर्चुअल मशीन](/virtual_machine/) होते हैं।

Comment (jayesh-srivastava) [inline content/hi/bare_metal_machine.md:9]: - क्योंकि कई सर्वर, [वर्चुअल मशीन](/virtual_machine/) होते हैं।

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

**Comment by anubhav06** [comment]
Hey. Sorry for the delay in completing the requested changes. I have my exams going on right now. Will get back to this in about 15 days.

**Comment by anubha-v-ardhan** [comment]
> Hey. Sorry for the delay in completing the requested changes. I have my exams going on right now. Will get back to this in about 15 days.

@anubhav06 Do we have any update?

File: content/hi/bare_metal_machine.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: बेयर मेटल मशीन
```
Comment (anubha-v-ardhan) [inline content/hi/bare_metal_machine.md:2]:
```suggestion
title: बेयर मेटल मशीन (Bare Metal Machine)
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by Garima-Negi — APPROVED** [review]

**Review by jayesh-srivastava — APPROVED** [review]
lgtm

**Review by anubha-v-ardhan — APPROVED** [review]
Thank you @anubhav06 
LGTM!

**Pull request merged**